storybook/storybook-docs/src/typography.stories.mdx

```diff
@@ -1,84 +1,160 @@
-import { Meta } from "@storybook/blocks";
+import { Meta, Typeset } from "@storybook/blocks";
 
 <Meta title="Docs/Designrichtlijnen/Typografie" />
 
 # Typografie
 
 ## Lettertype
 
-We gebruiken het lettertype Amsterdam Sans.
-De _fallback_ lettertypes zijn Arial en sans-serif.
+De gemeente gebruikt de Amsterdam Sans als beeldbepalend lettertype.
```

Comment:
This change differs a bit from the [Schrijfwijzer guidelines](https://www.amsterdam.nl/schrijfwijzer/heldere-taal-basis-onze-huisstijl/). They use a lot of 'we' there, not 'de gemeente'. Imo, the original sentence was more simple and direct.